### DISCUSSION TITLE
LFX Mentorship '22 Summer Semester

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#632] [Announcements]
Hello everyone!

The LFX Mentorship '22 Summer semester is open now open for mentee applications: https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/02-Summer#accepted-projects

We have compressed the administration schedule to work around the LF All hands and KubeCon events this year. The semester is the same length as it has been in previous years.

[Mentees information](https://docs.linuxfoundation.org/lfx/mentorship/mentees): how to apply, stipends, etc.

## Timeline

**Note, this has been updated**

### Summer Term: June 6th - August 31st

- mentorships available on LFX Mentorship: May 8th, 2021 (no change, existing proposals still available for application)
- **proposals cutoff May 24th**
- applications open: May 9th - **May 29th**
- application review/admission decisions/HR paperwork: **May 31st - June 3rd**

### Mentorship duration - three months (**11 weeks** - full-time schedule)

- **June 6 (Week 1)**: Mentorship program begins with the initial work assignments
- July 15 (**End of Week 5**): Midterm mentee evaluations and first stipend payments
- August 31 (**End of Week 11**): Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals

[original email](https://lists.cncf.io/g/cncf-toc/message/6858)

**Comment by anutosh491** [comment]
Hello @nate-double-u , hope you're doing well , this is Anutosh here from India . I'm an open source enthusiast and I'm currently involved with communities based on numerical and symbolic computations/algorithms in math and physics like numpy, sympy, scipy. I am keen to take part in the LFX Mentorship program for the summer term and it would be my first interaction with any Linux Foundation based mentorship program (I've been a part of a few mentorship programs before)

Just wanted to clarify the timeline !

> mentorships available on LFX Mentorship: May 8th, 2022
> applications open: May 9th - May 24th

 Does this mean we should be able to see all projects on the [Mentorship/Application portal](https://mentorship.lfx.linuxfoundation.org/#projects_accepting) and apply to then by May 9th which is today ?

As the readme points out , we should be using the Discussions tab to ask doubts .Hence, I hope this is the correct place to ask such a question !

**Reply by nate-double-u** [reply]
Hi @anutosh491, thanks for your interest!

I've just merged the last couple project submissions in, and I'm sitting down now to add them to the LFX Mentorship system.

My goal is to get them all into the system today, and I'll make an announcement when applications are opened.

**Reply by anutosh491** [reply]
Thanks for the reply , glad to hear from you . I wasn't sure whether the dates mentioned on the readme page were strict or tentative which led to the doubt above.
Will start working on my application as soon as the applications are out !

**Comment by nate-double-u** [comment]
Hello everyone!

We are now accepting mentee applications for the LFX Mentorship Summer '22 semester. Please check out the projects here:
https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/02-Summer#accepted-projects

Applications are open for two weeks: May 9th - May 24th

**Comment by Omkar0114** [comment]
Hey @nate-double-u, I'm omkar. I have some experience in Open source and now learning DevOps. I want to apply for the LFX mentorship program but I'm just learning new things, what should I do for the LFX mentorship program I'm really excited about such programs and I love to contribute to awesome projects :grinning: 
Can anyone guide me in this?

The application just opened,  so what should I do exactly in a minimum amount of time?

**Reply by nate-double-u** [reply]
Hi @Omkar0114, I suggest [looking through the projects](https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/02-Summer#accepted-projects), see what is interesting to you, and if there are any issues marked "good first issue" maybe take a stab at them. Communicate with project mentors; this will help both you and the projects to get a sense of what is needed.

If you're interested in something, go to [LFX Mentorship](https://mentorship.lfx.linuxfoundation.org) and create an account. Once you've done that, you can apply for a mentorship (all the projects in the list have an LFX URL to help you find them).

**Reply by nate-double-u** [reply]
I should also share the [Mentees docs page](https://docs.linuxfoundation.org/lfx/mentorship/mentees). This has information about how to apply, stipends, and more.

**Reply by Omkar0114** [reply]
Okay. Thanks a lot for helping me out! I'll go through what you mentioned above and really its helpful @nate-double-u 
I will start working on my application.

**Comment by Taiwrash** [comment]
Hi @nate-double-u ,

I am a CNCF community organizer and I am looking for mentorship on project that will help me community grow as a student. I  have experience writing c++ bit more strong in JavaScript frameworks React and presently exploring machine learning and DevOps. What project will you suggest for me to apply to, because I am more than Interested in this program.

**Reply by nate-double-u** [reply]
Hi @Taiwrash , I suggest [looking through the projects](https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/02-Summer#accepted-projects) and seeing what is interesting to you. Each has suggested skills, so hopefully, something there will match your skillset. If you see something interesting, communicate with project mentors; this will help both you and the projects get a sense of what is needed.

**Comment by nate-double-u** [comment]
Hello everyone!

We have updated the schedule to add more time for projects to add proposals, and extend the application deadline for mentees.

This will push back the program starte date, applications open timeline, and the application review and admissions decisions key dates. However, we will not be extending the end of the semester past August 31st.

## Updated Timeline

### Summer Term: June 6th - August 31st

- mentorships available on LFX Mentorship: May 8th, 2021 (no change, existing proposals still available for applicaiton)
- **proposals cutoff May 24th**
- applications open: May 9th - **May 29th**
- application review/admission decisions/HR paperwork: **May 31st - June 3rd**

### Mentorship duration - three months (**11 weeks** - full-time schedule)

- **June 6 (Week 1)**: Mentorship program begins with the initial work assignments
- July 15 (**End of Week 5**): Midterm mentee evaluations and first stipend payments
- August 31 (**End of Week 11**): Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals

**Reply by ToLToL** [reply]
Hi @nate-double-u,

We should update the timeline in https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/02-Summer as well.

**Reply by nate-double-u** [reply]
Good suggestion, thanks @ToLToL!

**Reply by SAEb-ai** [reply]
> Summer Term: June 6th - August 31st
> 
> - mentorships available on LFX Mentorship: May 8th, 2021 (no change, existing proposals still available for applicaiton)
> - **proposals cutoff May 24th**
> - applications open: May 9th - May 29th
> - application review/admission decisions/HR paperwork: May 31st - June 3rd

Hi @nate-double-u What is meant by the above  **proposals cutoff May 24th**? Thanks!!

**Reply by nate-double-u** [reply]
Hi @SAEb-ai, project maintainers have until May 24th to submit proposals for the summer 22 LFX Mentorship term.

**Reply by SAEb-ai** [reply]
Thanks @nate-double-u!! Although one more question. Suppose a mentee have applied to a project before 24th May, 2022 but haven't submitted their pre-requisite tasks till then. So, can they submit their pending tasks even after 24th May but till 29th May according to the updated timelines?

**Reply by nate-double-u** [reply]
Mentee applications are open till May 29th -- the may 24th proposal cutoff doesn't affect mentee applications.

Mentees need to have everything in by May 29th to have a complete application for consideration.

**Comment by dhavalgshah** [comment]
Hi @nate-double-u , are LFX mentorships open for career changers?

**Reply by nate-double-u** [reply]
Hi @dhavalgshah, please see the [eligibility requirements](https://docs.linuxfoundation.org/lfx/mentorship/mentee-guide/am-i-eligible) in the docs.

In your case, I think the last point is the one most likely to apply, that you:
> Not be a maintainer, recurring contributor, etc. with more than minimal involvement with the open source project that offers a mentorship program.

I hope that this is helpful!

**Comment by redoC-A2k** [comment]
Hello everyone ! I am new to cncf , **Do I have any chance of getting selected for June - August lfx mentorship now** ? Or I am very late . I am not familiar with kubernetes and GO much , But I am familiar with linux , docker , rust, python and also have done open source contributions to some gtk applications of gnome and also worked in MERN stack in the past . I hope, I have some chance for selection which are the projects I should focus on and how should I move forward after applying to projects ? Any help will be very helpful to me .. THANKS

**Reply by nate-double-u** [reply]
Hi @redoC-A2k, the applications are open till May 29th, plenty of time to apply still! 

As [I mentioned above](https://github.com/cncf/mentoring/discussions/632#discussioncomment-2765242): I suggest [looking through the projects](https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/02-Summer#accepted-projects) and seeing what is interesting to you. Each has suggested skills, so hopefully, something there will match your skillset. If you see something interesting, communicate with project mentors; this will help both you and the projects get a sense of what is needed.

**Reply by anutosh491** [reply]
Hello @nate-double-u  . Sorry for the silly doubt I'm about to ask but just to clarify , what do you mean by "till May 29th" (cause there is no indication about a time based limit for applying) . It is including May 29th (before 30th May starts) or excluding May 29th (before May 29th starts) ?!

I'm anyways ready with my resume and cover letter so shouldn't be an issue but still don't want to miss out on this wonderful oppurtunity :D

**Reply by nate-double-u** [reply]
Sorry, you're right, _till May 29th_ is somewhat vague; I intended to include the whole day of May 29th when I set it as a deadline, and I try to be as inclusive of time zones as I can be as I didn't set a time with the date. So I'm happy to accept an application as long as it's still May 29th somewhere in the world.

If you haven't applied yet, there is still some time!
